src/parsers/manifestjson.js

```diff
@@ -91,8 +91,15 @@ export default class ManifestJSONParser {
   errorLookup(error) {
     // This is the default message.
     var baseObject = messages.MANIFEST_JSON_INVALID;
+
+    // This is the default from webextension-manifest-schema, but it's not a
+    // super helpful error. We'll tidy it up a bit:
+    if (error.message === 'should NOT have additional properties') {
+      error.message = 'is not a valid key or has invalid extra properties';
+    }
```

Comment:
Shouldn't that actually be in `web-extension-manifest-schema`? I mean, once some decides to write the `NOT` in regular lower case characters this line is invalid an we won't even notice it. Well, the test might fail but still. Or, we should have error ids in the schema too to make targeting error message rewrites like that better but I'm more for just changing the error message in the schema than overwriting it here.